### PR TITLE
NNDSVD

### PR DESCRIPTION
**Pull request by tedmoore**

### CODE REVIEW

**Review by jamesb93 — APPROVED** [review]
I made some minor changes to make |buffer| be enclosed in the token for rendering.

Other than that its good.

**Comment by tremblap** [comment]
we need to change the name to FluidBufNMFSeed - we might as well do that here? or do you want me to review that, we merge than change it in SC and Doc before the max translation?

**Comment by jamesb93** [comment]
> we need to change the name to FluidBufNMFSeed - we might as well do that here? or do you want me to review that, we merge than change it in SC and Doc before the max translation?

I think we should get the shape of the documentation right and then apply a considered name change across all the CCEs in one blow.

**Comment by tremblap** [comment]
ok I'll review as is now but we should change the name before @jamesb93 translates to Max to avoid double work

File: doc/BufNNDSVD.rst [hunk 1]
```diff
@@ -3,50 +3,61 @@
 :sc-categories: FluidManipulation
 :sc-related: Classes/FluidBufNMF
 :see-also: 
-:description: 
-   Find Initial Bases and Activations for FluidBufNMF via Non-Negative Double Singular Value Decomposition .
-
-   See http://nimfa.biolab.si/nimfa.methods.seeding.nndsvd.html
-
+:description: Find Initial Bases and Activations for BufNMF
```
Comment (tremblap) [inline doc/BufNNDSVD.rst:6]: there is also the number of components. so we could say somethng a little more generic - computes seeding values for nmf components, bases and activations

**Review by tremblap — CHANGES_REQUESTED** [review]
this is great - a bit mathy but still I understand it better.

**Pull request merged**